modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/prepare/PrepareServiceImplTest.java

```diff
@@ -103,18 +118,158 @@ void prepareServiceCachesPlanCreatedForExplain() {
         assertThat(plan.plan(), sameInstance(queryPlan));
     }
 
+    @Test
+    void prepareReturnsQueryPlanThatDependsOnParameterTypeMatchInferred() {
```

Comment:
```suggestion
    public void prepareReturnsQueryPlanThatDependsOnParameterTypeMatchInferred() {
```

Reply:
For some reason all test cases in this class are package private .

Reply:
Fixed.